### DISCUSSION TITLE
Application not accepted for students developer pack

### DISCUSSION THREAD

**Discussion by apoorvpandey0** [community/community#23518] [New to GitHub]
<p>I’m frustrated,<br>
I have submitted a certificate from my College institute stating my admission in the institution to and from year, still my application is not accepted!<br>
I have tried my college ID, Enrollment form, Admission slip everything!!!<br>
What the hell do these guys want.</p>

**Comment by queenofcorgis** [comment, marked as answer]
<p>Hi <a class="mention" href="/apoorvpandey0">@apoorvpandey0</a> - apologies for the delay, this post was in our uncategorized so it received less visibility than it would if it were assigned to a category.</p>
<p>For student benefits and related conversations, we recommend visiting the <a href="https://github.com/orgs/github-community/discussions/9">Student Developer Pack FAQs post</a> as that may help answer your questions. If not, please use that Discussion to ask for assistance as that is the Education Community’s new home.</p>
